api/billingd/main.go

```diff
@@ -64,14 +68,6 @@ var (
 				Key:      "stripe.webhook_secret",
 				DefValue: "",
 			},
-			"segmentApiKey": {
-				Key:      "segment.api_key",
-				DefValue: "",
-			},
-			"segmentPrefix": {
-				Key:      "segment.prefix",
-				DefValue: "",
-			},
```

Comment:
@textileben and here are some `billingd` config options that are now gone (moved to `analyticsd`)